src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj

```diff
@@ -61,7 +61,4 @@ New features
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.1' ">
     <PackageReference Include="Newtonsoft.Json" Version="[9.0.1,12)" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
Why did you remove this package reference? This was added for some reason but I can't recall.